Debian/8/input/profiles/anssi_np_nt28_average.xml

```diff
@@ -0,0 +1,56 @@
+<Profile id="anssi_np_nt28_average">
+<title>Common Profile for ANSSI DAT-NT28 level average (fr: intermediaire) for Debian Systems</title>
```

Comment:
@pthierry38 Wouldn't it make sense to say this for a title for consistency in this PR:
`Profile for ANSSI DAT-NT28 level average (intermediate)`

Does `for Debian systems` really need to be in the title?